apps/filebrowser/src/filebrowser/templates/listdir_components.mako

```diff
@@ -1449,6 +1451,18 @@ else:
           $('#newRepFactorInput').val(self.selectedFile().stats.replication);
         });
       };
+      
+      self.allowCopyMoveTo = function (destination) {
+        const removeLastSlash = function (path) {
```

Comment:
Nit: Move up one level as more of a util function and doesn't depend on the args

Reply:
Yes, I placed it outside the allowCopyMoveTo function first, but then thought I should "contain" it a bit since the file is not really structured around clean functions. I can move it back up.